fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoansApiResourceSwagger.java

```diff
@@ -631,6 +631,8 @@ private GetLoansLoanIdLoanTransactionEnumData() {}
                 public boolean refundForActiveLoans;
                 @Schema(example = "false")
                 public boolean creditBalanceRefund;
+                @Schema(example = "false")
```

Comment:
I believe this field is not needed anymore.

Reply:
Done